src/altinn-studio-cli/Upgrade/Backend/v7Tov8/AppSettingsRewriter/AppSettingsRewriter.cs

```diff
@@ -89,36 +89,48 @@ private void RewriteRemoveHiddenDataSetting(string fileName, JsonObject settings
         }
 
         // Look for "RemoveHiddenDataPreview" property
-        appSettingsObject.TryGetPropertyValue("RemoveHiddenDataPreview", out var removeHiddenDataPreviewNode);
-        if (removeHiddenDataPreviewNode is not JsonValue removeHiddenDataPreviewValue)
+        try
         {
-            // No "RemoveHiddenDataPreview" property found, nothing to change
-            return;
-        }
+            appSettingsObject.TryGetPropertyValue("RemoveHiddenDataPreview", out var removeHiddenDataPreviewNode);
```

Comment:
Tested this, it still fails on line 84, that is where the error occurs:
```
Unhandled exception: System.ArgumentException: An item with the same key has already been added. Key: FeatureManagement (Parameter 'propertyName')
   at System.Text.Json.ThrowHelper.ThrowArgumentException_DuplicateKey(String paramName, String propertyName)
   at System.Text.Json.Nodes.JsonObject.InitializeDictionary()
   at System.Text.Json.Nodes.JsonObject.TryGetPropertyValue(String propertyName, JsonNode& jsonNode)
   at Altinn.Studio.Cli.Upgrade.Backend.v7Tov8.AppSettingsRewriter.AppSettingsRewriter.RewriteRemoveHiddenDataSetting(String fileName, JsonObject settings) in /_/src/altinn-studio-cli/Upgrade/Backend/v7Tov8/AppSettingsRewriter/AppSettingsRewriter.cs:line 84
   at Altinn.Studio.Cli.Upgrade.Backend.v7Tov8.AppSettingsRewriter.AppSettingsRewriter.Upgrade() in /_/src/altinn-studio-cli/Upgrade/Backend/v7Tov8/AppSettingsRewriter/AppSettingsRewriter.cs:line 58
   at Altinn.Studio.Cli.Upgrade.Backend.v7Tov8.BackendUpgrade.BackendUpgrade.UpgradeAppSettings(String appSettingsFolder) in /_/src/altinn-studio-cli/Upgrade/Backend/v7Tov8/BackendUpgrade/BackendUpgrade.cs:line 257
   at Altinn.Studio.Cli.Upgrade.Backend.v7Tov8.BackendUpgrade.BackendUpgrade.<>c__DisplayClass0_0.<<GetUpgradeCommand>b__10>d.MoveNext() in /_/src/altinn-studio-cli/Upgrade/Backend/v7Tov8/BackendUpgrade/BackendUpgrade.cs:line 134

```

Reply:
Sorry, tested in a nested object and got a error on another line 🙈 should be fixed now